utilities/llmd_utils.py

```diff
@@ -284,7 +284,9 @@ def create_llmisvc(
             template_config["imagePullSecrets"] = [{"name": secret} for secret in image_pull_secrets]
 
     if enable_auth:
-        annotations["serving.kserve.io/auth"] = "true"
+        annotations["security.opendatahub.io/enable-auth"] = "true"
```

Comment:
will this not affect the old test for serverless/raw?

Reply:
this is in the `create_llmisvc` function so it won't affect regular isvcs